hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/webapp/MockDefaultRequestInterceptorREST.java

```diff
@@ -279,4 +271,28 @@ public ContainersInfo getContainers(HttpServletRequest req, HttpServletResponse
 
     return containers;
   }
+
+  @Override
+  public NodeToLabelsInfo getNodeToLabels(HttpServletRequest hsr) throws IOException {
+    if (!isRunning) {
+      throw new RuntimeException("RM is stopped");
+    }
+    String cpuLabel = "CPU";
+    HashSet<String> cpuLabels = new HashSet<>();
+    cpuLabels.add(cpuLabel);
+
+    String gpuLabel = "GPU";
+    HashSet<String> gpuLabels = new HashSet<>();
```

Comment:
I think we have hash constructors that already take the default values at construction instead of having to add.

Reply:
Thanks for your suggestion, I will modify the code.